### DISCUSSION TITLE
Issue to install ffmpeg

### DISCUSSION THREAD

**Discussion by sorintodys** [savonet/liquidsoap#2249] [Installation]
Hello,

Is any way to solve this on debian 10?
```
#  | Requested 'libavutil >= 55.78.100' but version of libavutil is 55.28.100
#  | Requested 'libavcodec >= 57.107.100' but version of libavcodec is 57.48.101
# Fatal error: exception Failure("Requested 'libavutil >= 55.78.100' but version of libavutil is 55.28.100\nRequested 'libavcodec >= 57.107.100' but version of libavcodec is 57.48.101\n")
```
Thank you very much for help!

**Comment by toots** [comment]
Hi! I believe that the ffmpeg packages on debian 10 are outdated. You'd need to either move to a more recent Debian release or else find a backport of the most recent ffmpeg packages. Or compile it yourself.
